 <testsuites name="" tests="9" failures="1" time="">
   <properties>
     <property name="go.version" value="1.14"></property>
   </properties>
   <testsuite tests="9" failures="1" time="" name="github.com/kubebuilder/kuttl/pkg/version">
     <testcase classname="pkg1.test.test_things" name="test_params_func:2" time="" assertions="0">
       <failure message="test failure" type="">Traceback (most recent call last):&#xA;  File &#34;nose2/plugins/loader/parameters.py&#34;, line 162, in func&#xA;    return obj(*argSet)&#xA;  File &#34;nose2/tests/functional/support/scenario/tests_in_package/pkg1/test/test_things.py&#34;, line 64, in test_params_func&#xA;    assert a == 1&#xA;AssertionError</failure>
     </testcase>
   </testsuite>
 </testsuites>